.eslintignore

```diff
@@ -1,2 +1,2 @@
 scripts
-packages/get-alephium
\ No newline at end of file
+packages/get-extension-wallet
```

Comment:
Is this on purpose to ignore the lint for the package?

Reply:
Great catch, this was used during development, now it can be removed :+1: